packages/hardhat-core/test/internal/hardhat-network/provider/modules/eth/hardforks.ts

```diff
@@ -1886,28 +1886,44 @@ describe("Eth module - hardfork dependant tests", function () {
         assert.isDefined(block.excessBlobGas);
       });
 
-      it("should have a non empty parentBeaconBlockRoot in the genesis block and the value should be an expected one", async function () {
+      it("should have 0x0 as parentBeaconBlockRoot in the genesis block", async function () {
```

Comment:
This fixes a bug in the Hardhat implementation of Cancun support.

According to the [EIP](https://eips.ethereum.org/EIPS/eip-4788), the value should be zero in genesis blocks:
> If this EIP is active in a genesis block, the genesis header’s parent_beacon_block_root must be 0x0 and no system transaction may occur.